src/tink/domspec/Attributes.hx

```diff
@@ -28,10 +28,10 @@ typedef FieldSetAttr = {>GlobalAttr<Style>,
 }
 
 typedef TimeAttr = {>GlobalAttr<Style>,
-  @:optional var datetime:DateTime;
+  @:optional var dateTime:DateTime;
 }
 
-abstract DateTime(String) {
+abstract DateTime(String) from String {
```

Comment:
Not a big fan of the `from String` here to be honest, as it allows you to write nonsense like `<time dateTime="orange" />`.

Reply:
I like oranges :) I've removed the `from String` clause, and replaced it by an implicit `@:from Float` cast.